_ml-commons-plugin/api.md

```diff
@@ -0,0 +1,968 @@
+                ]
+            },
+            {
+                "values": [
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 1.00
+                    },
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 4.00
+                    }
+                ]
+            },
+            {
+                "values": [
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 1.00
+                    },
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 0.00
+                    }
+                ]
+            },
+            {
+                "values": [
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 10.00
+                    },
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 2.00
+                    }
+                ]
+            },
+            {
+                "values": [
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 10.00
+                    },
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 4.00
+                    }
+                ]
+            },
+            {
+                "values": [
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 10.00
+                    },
+                    {
+                        "column_type": "DOUBLE",
+                        "value": 0.00
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+
+
+### Response
+
+**Response for index data**
+
+```json
+{
+  "status" : "COMPLETED",
+  "prediction_result" : {
+    "column_metas" : [
+      {
+        "name" : "ClusterID",
+        "column_type" : "INTEGER"
+      }
+    ],
+    "rows" : [
+      {
+        "values" : [
+          {
+            "column_type" : "INTEGER",
+            "value" : 0
+          }
+        ]
+      },
+      {
+        "values" : [
+          {
+            "column_type" : "INTEGER",
+            "value" : 0
+          }
+        ]
+      },
+      {
+        "values" : [
+          {
+            "column_type" : "INTEGER",
+            "value" : 0
+          }
+        ]
+      },
+      {
+        "values" : [
+          {
+            "column_type" : "INTEGER",
+            "value" : 0
+          }
+        ]
+      },
+      {
+        "values" : [
+          {
+            "column_type" : "INTEGER",
+            "value" : 0
+          }
+        ]
+      },
+      {
+        "values" : [
+          {
+            "column_type" : "INTEGER",
+            "value" : 0
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+**Response for data input directly**
```

Comment:
Actually response for index data or direct input data are the same. This response for data input directly seems wrong. How about keep the response above only for both index data and direct data cases?

Reply:
Noticing the similarities. I tested with both FIT-RCF and Kmeans, and the inputs are similar. I'll keep the index data example.